### DISCUSSION TITLE
Cumulate updates failing on Lumia 950 XL (Download Error - 0x80070005)

### DISCUSSION THREAD

**Discussion by sreegitgub** [WOA-Project/Community-Issues#4] [Q&A]
I've win 11 build .132 and trying to do cumulative update to .160 , its not happening.
I've disabled dual boot still updates failing with  download error - 0x80070005

**Comment by gus33000** [comment]
winerror.h: 0x80070005
E_ACCESSDENIED
General access denied error

This sounds weird as it shouldn't happen with dual boot being disabled. (+ it's a download error)

Out of curiosity have you tried letting the update download with the screen always on?

**Comment by sreegitgub** [comment]
Yes , screen is always on while it’s trying to download.

On Sat, 21 Aug 2021 at 1:35 PM, Gustave Monce ***@***.***>
wrote:

> winerror.h: 0x80070005
> E_ACCESSDENIED
> General access denied error
>
> This sounds weird as it shouldn't happen with dual boot being disabled. (+
> it's a download error)
>
> Out of curiosity have you tried letting the update download with the
> screen always on?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/WOA-Project/Community-Issues/discussions/4#discussioncomment-1215530>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVJDIJ7SQGGWHEIGYGETYPTT556QPANCNFSM5CRRFJRQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>
-- 
Thanks & Regards,
Sreekanth Valluru

**Comment by MagicAndre1981** [comment]
Download the 3 updates on your own:

https://my.hidrive.com/lnk/aqvogzWf#file
https://my.hidrive.com/lnk/0mPoAdx2#file
https://my.hidrive.com/lnk/UuPoAJmf#file

extract [this script](https://github.com/abbodi1406/WHD/raw/master/scripts/W10UI_10.9.zip) in same folder and run Win10UI.cmd as admin (rightclick on cmd and select run as admin)

**Comment by sreegitgub** [comment]
is this work for win 11 cumulative update?

I am trying to update win 11 build .132 to 160

On Thu, Aug 26, 2021 at 8:47 AM MagicAndre1981 ***@***.***>
wrote:

> Download the 3 updates on your own:
>
> https://my.hidrive.com/lnk/aqvogzWf#file
> https://my.hidrive.com/lnk/0mPoAdx2#file
> https://my.hidrive.com/lnk/UuPoAJmf#file
>
> extract this script
> <https://github.com/abbodi1406/WHD/raw/master/scripts/W10UI_10.9.zip> in
> same folder and run Win10UI.cmd as admin (rightclick on cmd and select run
> as admin)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/WOA-Project/Community-Issues/discussions/4#discussioncomment-1237275>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVJDIJY6WDC6JDZFR54NLQ3T6XIO7ANCNFSM5CRRFJRQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
Thanks & Regards,
Sreekanth Valluru

**Reply by MagicAndre1981** [reply]
yes, this is for Win11. HEre you need the cab + psf file and use the script which extracts the psf.

here are the files for .168:

https://my.hidrive.com/lnk/XHvogj4t#file
https://my.hidrive.com/lnk/uPPIgEve#file
https://my.hidrive.com/lnk/5DvIAn1M#file

and here the latest script: https://github.com/abbodi1406/WHD/blob/16a32104e7c55349d868db2b3f0b81efedeb2d5d/scripts/W10UI_10.9f.zip?raw=true
